templates/template_index.html

```diff
@@ -454,6 +454,15 @@ <h2 id="compatibility" class="mb-4 text-center">Compatibility</h2>
   roughness: 0,
 });
 
+var xbox_ring_material = new THREE.MeshStandardMaterial( {
+  color: xbox_base_color,
+  depthTest: false,
+  roughness: 0,
+});
+
+const date = new Date();
+const frag = date.getMonth() === 3 && date.getDate() === 1;
```

Comment:
nit: `date.getMonth() === 4`

Reply:
`getMonth()` returns 0-11 to better suite an array with month-names.

Reply:
You would think... but months of getMonth start at 0

Reply:
`getDay()` does. :)

Reply:
> `getDay()` does. :)

Lol.